src/ImageRunModal.jsx

```diff
@@ -837,8 +837,19 @@ export class ImageRunModal extends React.Component {
                             </Flex>
                         </FormGroup>
 
-                        {(this.state.owner === systemOwner || (this.state.image && this.state.image.isSystem)) &&
-                            <FormGroup fieldId='run-image-cpu-priority' label={_("CPU shares")}>
+                        {this.state.owner === systemOwner &&
```

Comment:
Did you deliberately change the condition here?

Reply:
Yes, for the new create container dialog the cpu shares option was not shown.

Reply:
Ah woops, I put it in the wrong commit :)